### ISSUE TITLE
Still active?

### ISSUE THREAD

**Issue by sualko** [RemindMe/RemindMe#1]
@RemindMe in 2 minutes about this issue :stuck_out_tongue_closed_eyes:

**Comment by sualko** [comment]
Can we renew this service? I think it's a great idea.

**Comment by AudriusButkevicius** [comment]
@Qix- @RemindMe  can I atleast have the username and run it on your behalf or something like that?

**Comment by Qix-** [comment]
There was never anything to run. I might start the project again soon as I've needed this several times.

**Comment by AudriusButkevicius** [comment]
Right, if you don't have time, I am wiling to give a new home to the username and implement it.

**Comment by sualko** [comment]
I would also love to help.

**Comment by AudriusButkevicius** [comment]
I guess I can also do the implementation to prove that I am serious about this but I want to have confidence/acknoledgement my effort won't go to waste.

**Comment by Justinzobel** [comment]
I went through a few alternatives and found https://github.com/PromptMeBot is available.

**Comment by Qix-** [comment]
If someone has a serious implementation that works I'm very happy to hand this name off to them. Just comment here with a link to the repo and I'll get the ball rolling. :)

**Comment by Justinzobel** [comment]
> @Justinzobel
> 
> > https://github.com/PromptMeBot
> 
> It's 404.

Yes, meaning the name is available...
